_includes/post_includes/2022-02-porownanie-jezykow-cloud-functions/coldstarts.html

```diff
@@ -0,0 +1,360 @@
+<div>
+    <button class="tablinks coldstarts" onclick="showColdstartTable(event, 'coldstarts1')" id="defaultOpenColdstarts">Run #1</button>
+    <button class="tablinks coldstarts" onclick="showColdstartTable(event, 'coldstarts2')">Run #2</button>
+</div>
+
+<div id="coldstarts1" class="tabcontent coldstarts">
```

Comment:
Super są te tabelki. Jedyne uwaga, że strzałki do sortowania nakładają się na etykietę kolumny.

Reply:
strzałki nakładają się bo niestety już się nie mieszczą, nic na to nie poradzę, dobrze że tabelka się zmieściła ledwo i jest czytelna